Development-Guide/AsciiDoc-Guide/README.adoc

```diff
@@ -0,0 +1,140 @@
+
+= AsciiDoc
+
+== Headers
+
+Writing an equals sign before a line of text will make it into a header.
+
+----
+= A Descriptive Title
+----
+
+Equals signs can be added to give preceding header levels.
+
+----
+= h1
+
+== h2
+
+=== h3
+----
+
+== Lists
+
+=== Unordererd Lists
+
+An unordered list, ie. a list of bullet points can be made with asterisks.
+
+----
+* First Item
+* Second Item
+----
+
+Will appear as:
+
+* First Item
+* Second Item
+
+=== Numbered Lists
+
+A numbered list is done with periods.
+
+----
+. Step 1
+. Step 2
+----
+
+Will appear as:
+
+. Step 1
+. Step 2
+
+=== Nested Lists
+
+Nested steps can be added by increasing the number of periods before the step.
+
+----
+. step 1
+.. step 1a
+. step 2
+----
+
+Will appear as:
+
+. step 1
+.. step 1a
+. step 2
+
+== Source Code
+
+To write source code in asciidoc, write "[source,a language]"
+Lines after the square brackets will be displayed as a block of code.
+An empty line will end the block of code.
+For example:
+
+----
+[source, c]
+//Some c code
```

Comment:
Surrounding with `----` is our preferred way of doing it. Please show it as such.